multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/CreateOrUpdateAppStepTest.java

```diff
@@ -92,16 +92,19 @@ void testCreateApplicationFromDockerImage() {
                                                                                           .password("somePassword")
                                                                                           .build())
                                                    .build();
+        Staging staging = ImmutableStaging.builder()
```

Comment:
Maybe `dockerStaging` would be more descriptive

Reply:
Done